src/pretix/plugins/sendmail/views.py

```diff
@@ -36,18 +39,35 @@ def form_valid(self, form):
         self.request.event.log_action('pretix.plugins.sendmail.sent', user=self.request.user, data=dict(
             form.cleaned_data))
 
+        tz = pytz.timezone(self.request.event.settings.timezone)
+
         failures = []
         for o in orders:
-            try:
-                mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                     None, self.request.event, locale=o.locale, order=o)
-            except SendMailException:
-                failures.append(o.email)
-
-        if failures:
-            messages.error(self.request, _('Failed to send mails to the following users: {}'.format(' '.join(failures))))
-        else:
-            messages.success(self.request, _('Your message has been queued to be sent to the selected users.'))
+            if self.request.POST.get("action") == "preview":
+                message = self.request.POST.get('message_0')
```

Comment:
You can use ``form.cleaned_data['message']`` here, so you don't need to care about the ``_0`` stuff, wherever that is coming from